Jellyfin.Windows.Tray/TrayApplicationContext.cs

```diff
@@ -147,35 +149,45 @@ private void LoadJellyfinConfig()
         _configFile = Path.Combine(_dataFolder, "config\\system.xml").ToString();
         _networkFile = Path.Combine(_dataFolder, "config\\network.xml").ToString();
         _executableFile = Path.Combine(_installFolder, "jellyfin.exe");
-        _port = "8096";
 
         if (File.Exists(_configFile))
         {
             XDocument systemXml = XDocument.Load(_configFile);
             XPathNavigator settingsReader = systemXml.CreateNavigator();
 
             _firstRunDone = settingsReader.SelectSingleNode("/ServerConfiguration/IsStartupWizardCompleted").ValueAsBoolean;
-            var publicPort = settingsReader.SelectSingleNode("/ServerConfiguration/PublicPort")?.Value;
-            if (!string.IsNullOrEmpty(publicPort))
-            {
-                _port = publicPort;
-            }
+
         }
 
         if (File.Exists(_networkFile))
         {
             XDocument networkXml = XDocument.Load(_networkFile);
             XPathNavigator networkReader = networkXml.CreateNavigator();
 
+            _requireHTTPS = settingsReader.SelectSingleNode("/NetworkConfiguration/RequireHttps").ValueAsBoolean;
             _networkAddress = networkReader.SelectSingleNode("/NetworkConfiguration/LocalNetworkAddresses").Value;
+            _port = networkReader.SelectSingleNode("/NetworkConfiguration/PublicPort")?.Value;
+
+            if (_requireHTTPS)
+            {
+                _uriScheme = "https://";
+                _networkAddress = networkReader.SelectSingleNode("/NetworkConfiguration/BaseUrl").Value;
```

Comment:
I'm pretty sure the `BaseUrl` is something like `/jellyfin`

Reply:
In my network.xml, it's blank, and this doesn't have anything set: https://cs.github.com/jellyfin/jellyfin/blob/499b1df9cb7b660b8fd6e0ac5e75226b2656da77/Jellyfin.Networking/Configuration/NetworkConfiguration.cs#L29

Reply:
Right- it's a user-configurable subpath.

As this PR stands the `_localJellyfinUrl` would be set to something like the following if a baseUrl is set `https:///jellyfin:8096/web/index.html`

Reply:
And now we know why this is a hobby, not a job. Will fix.